CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ x.x.x Release notes (yyyy-MM-dd)
   properties.
 * Indexing `BOOL`/`Bool` and `NSDate` properties are now supported.
 * Swift: Add support for indexing optional properties.
+* `NSPredicate`'s `SUBQUERY` operator is now supported. It has the following limitations:
+  * `@count` is the only operator that may be applied to the `SUBQUERY` expression.
```

Comment:
we should file GitHub issues tracking these limitations so we remember to address them eventually.